src/mpid/ch4/src/ch4_recv.h

```diff
@@ -208,8 +208,6 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_cancel_recv_unsafe(MPIR_Request * rreq)
   fn_exit:
     MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_CANCEL_RECV_UNSAFE);
     return mpi_errno;
-  fn_fail:
```

Comment:
Same as above; let's check the `mpi_errno` value. I know we don't much at `fn_fail`, but we can add error handling code there in the future